kernel-python/urth/util/serializers.py

```diff
@@ -26,7 +28,28 @@
     from urth.util.base_serializer_py3 import BaseSerializer
 
 
-class DataFrameSerializer(BaseSerializer):
+class PandasSeriesSerializer(BaseSerializer):
+    @staticmethod
+    def klass():
+        import pandas
+        return pandas.Series
+
+    @staticmethod
+    def serialize(obj, **kwargs):
+        # Default to index orientation
+        # {index -> {column -> value}}
+        date_format = kwargs.get('date_format', 'iso')
+        return json.loads(obj.to_json(orient='index', date_format=date_format))
```

Comment:
You might, except I chose to serialize the Series objects using the default `index` orientation, which basically gives you the equivalent of a `dict` in Python, or an `Object` in JavaScript.  So limiting in this sense is like saying, give me the first 100 attributes of the object.

I suppose if we wanted to change the default serialization to treat the data more like a long array, then the limit makes more sense.  

Either way, I can add the code to honor a limit.